cdap-ui/app/cdap/components/DataPrep/DataPrep.scss

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+.dataprep-container {
+  &.error { padding-top: 25px; }
+
+  .dataprep-body.row {
+    // height calculations
+    // 100vh - height of navbar - top panel - dataprep cli - footer
+    // 100vh - 50px - 50px - 50px - 54px
+    margin-right: 0;
+    margin-left: 0;
+    height: calc(100vh - 154px);
```

Comment:
my bad, forgot to update comment. The CLI is inside this, so didn't need to subtract the CLI height anymore